ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6605,4 +6605,26 @@ VMADC:
   - mixed_air_damper_percentage_command
   - mixed_air_damper_percentage_sensor
   implements:
-  - CONTROL
\ No newline at end of file
+  - CONTROL
+
+SEWTM:
+  description: "Secondary water temperature monitoring"
+  opt_uses:
+  - secondary_supply_water_temperature_sensor
+  uses:
+  - secondary_return_water_temperature_sensor
+  implements:
+  - MONITORING
+
+HWTTCX2:
+  description: "Hot water tank temperature control X2 with single setpoint"
+  is_abstract: true
+  opt_uses: 
+  - heater_run_command_1
+  - heater_run_command_2
+  - high_water_temperature_alarm_1
+  - high_water_temperature_alarm_2
+  uses:
+  - hot_water_tank_temperature_setpoint
```

Comment:
Should we also have `hot_water_tank_temperature_sensor_1` and `hot_water_tank_temperature_sensor_2`?

Reply:
Agree, please see `HWTTC` for precedent.